vms/proposervm/vm.go

```diff
@@ -305,14 +341,16 @@ func (vm *VM) SetPreference(ctx context.Context, preferred ids.ID) error {
 		childBlockHeight = blk.Height() + 1
 		parentTimestamp  = blk.Timestamp()
 		nextStartTime    time.Time
+		proposalSlot     = unspecifiedSlotIndex
 	)
 	if vm.IsDurangoActivated(parentTimestamp) {
 		currentTime := vm.Clock.Time().Truncate(time.Second)
+		proposalSlot = proposer.TimeToSlot(parentTimestamp, currentTime)
```

Comment:
This is the "first proposal slot to start looking for which proposal slot will be ours." Not our proposal slot. I think we actually want to set the metric to `proposer.TimeToSlot(parentTimestamp, nextStartTime)`. Right?